config/routes.rb

```diff
@@ -1,13 +1,13 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     root to: "pages#home"
-    get "pages/home"
+    get "home", to: "pages#home"
     resources :products
     get "login", to: "sessions#new"
     post "login", to: "sessions#create"
     delete "logout", to: "sessions#destroy"
     resource :cart
-
+    get "admin", to: "admin#index"
```

Comment:
Chú vẫn chưa tìm hiểu về namespace nhỉ
https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing